packages/manager/apps/telecom/src/app/telecom/telephony/repayments/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,21 @@
+{
+  "telephony_repayments_available_title": "Reversements disponibles",
+  "telephony_repayments_costs_title": "Coûts à verser",
+  "telephony_repayments_costs": "Coûts",
+  "telephony_repayments_grid_buyAmount.value": "Reversement",
+  "telephony_repayments_grid_calledNumber": "Numéro spécial",
+  "telephony_repayments_grid_callingNumber": "Numéro entrant",
+  "telephony_repayments_grid_duration": "Durée (sec.)",
+  "telephony_repayments_grid_startDate": "Date",
+  "telephony_repayments_grid_status_CREATED": "En cours de reversement",
+  "telephony_repayments_grid_status_ERROR": "Erreur sur le reversement",
+  "telephony_repayments_grid_status_NONE": "Aucun reversement",
+  "telephony_repayments_grid_status_PAID": "Reversé",
+  "telephony_repayments_grid_status": "Statut",
+  "telephony_repayments_iban_link": "Modifier mes coordonnées bancaires",
+  "telephony_repayments_message_above_threshold": "Votre prochain reversement sera de <strong>{{ amount }}</strong>",
```

Comment:
You can move the html `<strong>` element into `data-translate-values`.
Sample `data-translate-values={ amount: '<strong>'+ $ctrl.amount +'</strong>' }`

Reply:
It's applied